packages/wonder-blocks-timing/src/util/__tests__/interval.test.js

```diff
@@ -99,13 +104,18 @@ describe("Interval", () => {
     describe("#set", () => {
         it("should call setInterval", () => {
             // Arrange
-            const interval = new Interval(() => {}, 500);
+            const intervalSpy = jest.spyOn(global, "setInterval");
+            const interval = new Interval(
+                () => {},
+                500,
+                SchedulePolicy.OnDemand,
+            );
```

Comment:
Why the switch to `OnDemand` for this test?

Reply:
Because that was what the test was written to work with, but it didn't use it and so it fails.